### DISCUSSION TITLE
Unmasked value in v-model

### DISCUSSION THREAD

**Discussion by avxkim** [beholdr/maska#141] [Ideas]
Is it possible to get unmasked value in v-model? I don't see a reason why we would need masked value in a model.

**Comment by inNetto** [comment]
I dont think so .. just fixed with this .. 

```
const maskaOptions = {
  mask: '+1 (###) ###-##-##',
  onMaska: (detail: MaskaDetail) => console.log(detail.unmasked)
}

<v-text-field
        v-model="cpf"
        v-maska:[maskaOptions]
        label="Cpf"
        :error-messages="errors.cpf"
/>


```
The problem is: If i have more than one field to mask.... i need to build more than one object option

**Comment by jboada** [comment]
Hi @avxkim ,

You can find another way to get the unmasked value at [https://beholdr.github.io/maska/#/?id=programmatic-use](https://beholdr.github.io/maska/#/?id=programmatic-use) .
